autogpt_platform/backend/backend/api/features/onboarding_dump/quality_test.py

```
@@ -0,0 +1,290 @@
+
+# --- threshold seams: exact boundaries route correctly --------------------
+
+# 45 distinct natural words, sliced to sit exactly on CLEAR_PASS_WORDS.
+_DISTINCT_WORDS = (
+    "my bakery ships fresh sourdough daily while managing wholesale orders "
+    "refunds invoices suppliers deliveries roster payroll marketing newsletters "
+    "customers complaints reviews inventory flour butter yeast ovens repairs "
+    "budget taxes accounting spreadsheets emails scheduling drivers routes "
+    "packaging labels promotions seasonal catering events quarterly forecasts "
+    "vendors contracts"
+).split()
```

Ruff (0.16.1)

[warning] Consider using a list literal instead of `str.split`

Replace with list literal

(SIM905)